connect/models/product.py

```diff
@@ -27,16 +28,24 @@ def make_object(self, data):
 class Item(BaseModel):
     global_id = None  # type: str
     mpn = None  # type: str
-    old_quantity = None  # type: int
-    quantity = None  # type: int
+    old_quantity = None  # type: Union[int, None]
```

Comment:
what is the idea to keep model with string here?

Reply:
What do you mean?